types/react-router-dom/index.d.ts

```diff
@@ -45,19 +46,29 @@ export interface HashRouterProps {
 }
 export class HashRouter extends React.Component<HashRouterProps, any> {}
 
-export interface LinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
```

Comment:
The Husky commit hook made all these whitespace changes. Should I have just bypassed Husky?